routing_common/transit_types.cpp

```diff
@@ -237,12 +239,14 @@ Line::Line(LineId id, std::string const & number, std::string const & title,
 bool Line::IsEqualForTesting(Line const & line) const
 {
   return m_id == line.m_id && m_number == line.m_number && m_title == line.m_title &&
-         m_type == line.m_type && m_networkId == line.m_networkId && m_stopIds == line.m_stopIds;
+         m_color == line.m_color && m_type == line.m_type && m_networkId == line.m_networkId &&
```

Comment:
I suggest you to follow the order of declaration here, i.e. compare colors after types.

Reply:
Yes. It was a mistake. Thanks.